stripecardscan/src/main/java/com/stripe/android/stripecardscan/scanui/SimpleScanStateful.kt

```diff
@@ -4,7 +4,7 @@ import android.util.Log
 import com.stripe.android.camera.framework.AnalyzerLoopErrorListener
 import com.stripe.android.stripecardscan.framework.Config
 
-internal abstract class ScanState(val isFinal: Boolean)
+abstract class ScanState(val isFinal: Boolean)
```

Comment:
what makes this have to be public?

Reply:
The fragment is public so it exposes this

Reply:
Makes sense. let's tag this as `@RestrictTo` the library group